R/export_transect_evdata.R

```diff
@@ -54,11 +54,29 @@ export_transect_evdata <- function(prjdir, transectname, horizbin) {
                                  file.path(transect_dir, 'EpiLayerLine_Final.csv'))
     usethis::ui_done("Epi line exported.")
     }
+  
```

Comment:
I really like the addition of this section. I'm also planning on adding a code block that will export the regions.